simulation-ros/src/turtlebot2i/turtlebot2i_description/v-rep_model/NewScriptForDockStation

```diff
@@ -0,0 +1,46 @@
+--**************************
+-- Recharge Station
+-- @author Klaus Raizer
+-- @date 22-06-2016
+-- 
+-- Description: This script was added so that Recharge Stations's 
+--names addhere to the automatic naming standard of adding "# plus number 
+--of new object".
+--
+-- This script calls a lua script located at the same folder as this scene's with the name: logistics_multiple_robots_youBot.lua
+-- Enyu Cao edited it to hide the error messages durning intialization.
+--**************************
+
+if not __notFirst__ then
```

Comment:
should this file have a `.lua` extension?

Reply:
Hi @berezovskyi , yes, it is a lua file. But this code should be inside the model .ttt file. I wrote it here so that @albertohata  or @KRaizer  can update the script in the warehouse_turtlebot2i.ttt.